packages/opentelemetry-instrumentation-qdrant/pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[tool.coverage.run]
+branch = true
+source = [ "opentelemetry/instrumentation/qdrant" ]
+
+[tool.coverage.report]
+exclude_lines = [ "if TYPE_CHECKING:" ]
+show_missing = true
+
+[tool.poetry]
+name = "opentelemetry-instrumentation-qdrant"
+version = "0.10.1"
+description = "OpenTelemetry Qdrant instrumentation"
+authors = [
+  "Gal Kleinman <gal@traceloop.com>",
```

Comment:
:question: Should contributors that create an instrumentation put themselves as an author here? :thinking:

Reply:
I don't have a strong opinion here. In general I'm fine with putting the main author here. As long as the license is APACHE it doesn't affect or limit the usage of the instrumentation.